extensions/hearthstone/src/card-list.tsx

```diff
@@ -0,0 +1,275 @@
+import { Action, ActionPanel, Grid, Icon, showHUD } from '@raycast/api';
+import { useEffect, useState } from 'react';
+import { CardDetailView } from './components/card-detail-view';
+import { Card, ClassName } from './types/types';
+import { getLocalCardData } from './utils/utils';
+
+export default function CardListCommand() {
+  const [cards, setCards] = useState<Card[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [searchByCost, setSearchByCost] = useState<string>('all');
+  const [language, setLanguage] = useState<'enUS' | 'zhCN'>('enUS');
```

Comment:
Set the language to match the `defaultCardImageLanguage` preference. Additionally, add the `defaultValue` attribute to the dropdown to ensure the content language aligns with the user's selection.